src/Features/CSharp/Portable/EditAndContinue/SyntaxUtilities.cs

```diff
@@ -216,27 +216,22 @@ public static SyntaxNode TryGetEffectiveGetterBody(ArrowExpressionClauseSyntax p
 
         public static SyntaxTokenList? TryGetFieldOrPropertyModifiers(SyntaxNode node)
         {
-            if (node.IsKind(SyntaxKind.FieldDeclaration))
```

Comment:
I wish the team preference of braces/no-braces/relaxed for single line if statements can be enforced on CI, so I don't have to ask every time what is our preference here.